src/routes/namespaces/index.svelte

```diff
@@ -1,5 +1,7 @@
-<script context="module" lang="ts">
-  export async function load() {
-    return { status: 302, redirect: '/namespaces/default' };
-  }
+<script lang="ts">
+  import { settings } from '$lib/stores/settings';
+  import { goto } from '$app/navigation';
+
+  const navigateTo = $settings.defaultNamespace;
+  goto(`/namespaces/${navigateTo}`);
```

Comment:
Does this add to the history?

ie.

Navigate to

namespaces ->
/namespaces/defaultNamespace

or will the history just be 
/namespaces/defaultNamespace

I think it should be the latter and we should double check.

If it's the first I think we can use the `replaceState` param on goto to support the second version.

Reply:
done